docs/docusaurus/docs/cloud/connect/connect_snowflake.md

```diff
@@ -0,0 +1,208 @@
+
+You'll need your access token and organization ID to set your access credentials. Don't commit your access credentials to your version control software.
+
+1. In GX Cloud, click **Settings** > **Tokens**.
+
+2. In the **User access tokens** pane, click **Create user access token**.
+
+3. In the **Token name** field, enter a name for the token that will help you quickly identify it.
+
+4. Click **Create**.
+
+5. Copy and then paste the user access token into a temporary file. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field into the temporary file with your access token and then save the file. 
+
+   GX recommends deleting the temporary file after you set the environment variables.
+
+## Deploy the GX Agent
+
+You deploy and run the GX Agent within your production cloud services environment. You can deploy the GX Agent container in any environment where you can run Docker container images or create Kubernetes clusters.
+
+To learn how to deploy a Docker container image in a specific environment, see the following documentation:
+
+- [Quickstart: Deploy a container instance in Azure using the Azure CLI](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-quickstart)
+
+- [Build and push a Docker image with Google Cloud Build](https://cloud.google.com/build/docs/build-push-docker-image)
+
+- [Deploy Docker Containers on Amazon ECS](https://aws.amazon.com/getting-started/hands-on/deploy-docker-containers/)
+
+You can deploy the GX Agent in any environment in which you create Kubernetes clusters. For example:
+
+- [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+- [Microsoft Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks-start-here)
+
+- [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/docs)
+
+- Any Kubernetes cluster version 1.21 or greater which uses standard Kubernetes
+
+<Tabs
+  groupId="connect-snowflake"
+  defaultValue='docker'
+  values={[
+  {label: 'Docker', value:'docker'},
+  {label: 'Kubernetes', value:'kubernetes'},
+  ]}>
+<TabItem value="docker">
+
+1. Download the GX Agent Docker container image from [Docker Hub](https://hub.docker.com/r/greatexpectations/agent).
+
+2. After configuring your cloud service to run Docker containers, run the following Docker command to start the GX Agent: 
+
+   ```bash title="Terminal input"
+   docker run -it \
+   -e GX_CLOUD_ACCESS_TOKEN= YOUR_ACCESS_TOKEN \ 
+   -e GX_CLOUD_ORGANIZATION_ID= YOUR_ORGANIZATION_ID \  
+   greatexpectations/agent:latest
+    ```
+    Replace `YOUR_ACCESS_TOKEN` and `YOUR_ORGANIZATION_ID` with the values you copied previously. To store your Snowflake password as an environment variable, add `-e GX_CLOUD_SNOWFLAKE_PW= YOUR_SNOWFLAKE_PASSWORD \` to the Docker command.
```

Comment:
If we are no longer passing in PW as part of the ENV, then the second sentence about `GX_CLOUD_SNOWFLAKE_PW` will be going away

Reply:
@Shinnnyshinshin I have clarified this is optional. I think it's helpful to include this information.